### ISSUE TITLE
clusterio crash

### ISSUE THREAD

**Issue by kevin28115** [Danielv123/factorioClusterio#70]
I just installed clusterio and was just trying to see how iti worked (haven't even figured out how to transfer items yet) and I had an error which crashed the server. attached is the error that I got. it happened when I put the  get tank via a bot. 
![clusterio error](https://user-images.githubusercontent.com/37888579/38066107-8d52d004-32d4-11e8-988e-8154dd65aeb9.PNG)

mods is also attached. 
[mods.zip](https://github.com/Danielv123/factorioClusterio/files/1858382/mods.zip)

**Comment by Danielv123** [comment]
Thanks for reporting, I will investigate.

**Comment by Danielv123** [comment]
Steps taken to reproduce:

* Downloaded all mods

* Downloaded factoio 0.16.36

* Built put and get tank in fresh save

* Build put and get tank in fresh save with robots

* Deconned put and get tank, both manually and with robots

I was not able to reproduce the crash. Does it only happen some times, or does it happen every time? Does other parts of the mod work? What version of factorio are you using?

If you are able to reproduce the crash, can you try disabling the other mods one by one until you have isolated the fault?

**Comment by kevin28115** [comment]
version 0.16.36

removed all mods other than clusterio and it still crashed when placing it. regardless if using bot or not.

did not crashed if purely using factorio without master and client. Only crashed when hosting factorio server via client.js

**Comment by Danielv123** [comment]
Hm, interesting. Will try again later.

**Comment by Danielv123** [comment]
Managed to reproduce now while running on a full instance.

```
 222.964 Error MainLoop.cpp:1029: Exception at tick 83163: Error while running event clusterio::on_tick (ID 0)
LuaEntity doesn't contain key recipe.
stack traceback:
	__clusterio__/control.lua:276: in function 'HandleOutputTanks'
	__clusterio__/control.lua:125: in function <__clusterio__/control.lua:103>
 222.965 Error ServerMultiplayerManager.cpp:96: MultiplayerManager failed: "Error while running event clusterio::on_tick (ID 0)
LuaEntity doesn't contain key recipe.
stack traceback:
	__clusterio__/control.lua:276: in function 'HandleOutputTanks'
	__clusterio__/control.lua:125: in function <__clusterio__/control.lua:103>"
 222.965 Info ServerMultiplayerManager.cpp:698: mapTick(83163) changing state from(InGame) to(Failed)
```

**Comment by Danielv123** [comment]
I believe I fixed this issue with version 1.5.3 https://github.com/Danielv123/factorioClusterioMod/releases/tag/1.5.3

If there are any more problems, reopen this issue or create a new one.
